cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/RoleManagementPrivilegeTest.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen.refcard
+
+class RoleManagementPrivilegeTest extends AdministrationCommandTestBase {
+  val title = "(★) ROLE MANAGEMENT PRIVILEGES"
+  override val linkId = "administration/security/administration/#administration-security-administration-dbms-privileges-role-management"
+
+  private def setup() = graph.withTx { tx =>
+    tx.execute("CREATE ROLE my_role")
+    tx.execute("DENY SHOW ROLE ON DBMS TO my_role")
+  }
+
+  def text: String = {
+    setup()
+    """
+###assertion=update-one
+//
+
+GRANT CREATE ROLE ON DBMS TO my_role
+###
+
+Grant the privilege to create roles to the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT DROP ROLE ON DBMS TO my_role
+###
+
+Grant the privilege to delete roles to the role `my_role`.
+
+###assertion=update-one
+//
+
+DENY ASSIGN ROLE ON DBMS TO my_role
+###
+
+Deny the privilege to assign roles to users to the role `my_role`.
+
+###assertion=update-one
+//
+
+DENY REMOVE ROLE ON DBMS TO my_role
+###
+
+Deny the privilege to remove roles from users to the role `my_role`.
+
+###assertion=update-one
+//
+
+REVOKE DENY SHOW ROLE ON DBMS FROM my_role
+###
+
+Revoke the existing denied privilege to show roles from the role `my_role`.
+
+###assertion=update-one
+//
+
+GRANT ROLE MANAGEMENT ON DBMS TO my_role
+###
+
+Grant the privileges to create, delete, assign, remove and show roles to the role `my_role`.
```

Comment:
As with `ALL ON DATABASE`, it might be a bit risky to list all of the privileges explicitly in case we add new ones and forget to update this lst. In this case we do not have that many privileges though and I think it is less risk that we will add more, so maybe ok to keep it here.